students/tfbanks/Lesson03/Mailroom_pt1.py

```diff
@@ -0,0 +1,120 @@
+# mailroom_pt1.py Exercise by tfbanks
+
+# !/usr/bin/env python3
+
+# Donor Database containing names and donations
+donor_db = [
+    ['Tim Cooker', 1500.50, 25050.50, 15680.75],
+    ['Elon Musket', 5250.25, 26500],
+    ['Frank Petersmankempt', 550.60],
+    ['Megan Morgan', 650.40, 1600, 20625.40],
+    ['Marlene Wheeler', 820, 1222.80]
+]
+
+donor = []  # Craate Donor List
```

Comment:
I would recommend calling this `donor_names` for readability and to conform to singular / plural naming for data structures.